packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -198,21 +198,21 @@ function NetworkContainer({
 
     if (isArbitrumNova) {
       return {
-        backgroundImage: `url('/images/ArbitrumNovaLogo.svg')`,
-        backgroundClassName: 'bg-[#8a4100]'
+        backgroundImage: `url('/images/TransparentNovaLogo.webp')`,
+        backgroundClassName: 'bg-orange-dark-nova'
       }
     }
 
     return {
-      backgroundImage: `url('/images/ArbitrumOneLogo.svg')`,
+      backgroundImage: `url('/images/TransparentArbitrumOneLogo.webp')`,
       backgroundClassName: 'bg-blue-arb-one'
     }
   }, [network])
 
   return (
-    <div className={`rounded-xl p-2 transition-colors ${backgroundClassName}`}>
```

Comment:
need to adjust the spacing in network option list
<img width="679" alt="image" src="https://user-images.githubusercontent.com/13184582/234030012-1cf2c3aa-09ba-44e1-9f97-174e21cc000c.png">

Reply:
Done - updated a bit to look more like design.